src/Sylius/Bundle/ApiBundle/Resources/config/services/command_handlers.xml

```diff
@@ -78,8 +78,8 @@
         <service id="Sylius\Bundle\ApiBundle\CommandHandler\Checkout\CompleteOrderHandler">
             <argument type="service" id="sylius.repository.order"/>
             <argument type="service" id="sm.factory" />
-            <argument type="service" id="sylius.mailer.order_email_manager" />
-            <tag name="messenger.message_handler" />
+            <argument type="service" id="sylius_event.bus" />
+            <tag name="messenger.message_handler" bus="sylius_default.bus" />
```

Comment:
I'm wondering about naming. Maybe we should have two busses:
* sylius.command_bus
* sylius.event_bus

and try to alias `sylius_default.bus` to `sylius.command_bus` if it is possilbe